README.md

```diff
@@ -954,24 +954,51 @@ Keep in mind, queue operations and management is an advanced discipline. This st
 
 ### Database connections
 
-Each GoodJob execution thread requires its own database connection that is automatically checked out from Rails’ connection pool. For example:
+GoodJob job executor processes require the following database connections:
```

Comment:
I can think the extra detail in this section overall is useful. 

And I think the level of detail implies something different than my general advice: 

Be generous with your database connection pool size. Better to configure a slightly larger pool than you think you need (which will only consume as many connections as it actually needs) than a smaller one (which will lead to connection pool timeouts and exceptions).

Reply:
> Be generous with your database connection pool size. Better to configure a slightly larger pool than you think you need (which will only consume as many connections as it actually needs) than a smaller one (which will lead to connection pool timeouts and exceptions).

I think this is good advice. My immediate follow-up question to that as a user of the gem is, "okay, well, then what's the lower bound where I'll start experiencing timeouts and exceptions, so I can make sure I always avoid that?"

Maybe that could be answered more concisely than what I am currently proposing.

The original docs answered that for `:async` mode, but lost me when I started thinking about `:external` mode. I think it was the lack of clarity between what a "job executor" process and its threads need and what a "job enqueuer" process and its threads need. Because those are the same process in `:async` mode (albeit with different thread pools for serving web requests and performing jobs), the requirements get jumbled together.